wiki/Tournaments/o!mAC/2019/en.md

```diff
@@ -0,0 +1,106 @@
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2019-08-11/2019-08-31 |
+| Qualifiers | 2019-09-21/2019-09-22 |
+| Live Drawings | 2019-09-23 21:00 (UTC+7) |
+| Group Stage | 2019-09-28/2019-10-06 |
+| Round of 16 | 2019-10-12/2019-10-13 |
+| Quarterfinals | 2019-10-19/2019-10-20 |
+| Semifinals | 2019-10-26/2019-10-27 |
+| Finals - Week 1 | 2019-11-02/2019-11-03 |
+| Finals - Week 2 | 2019-11-09/2019-11-10 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile Badge (To be confirmed), 4 month of osu!supporter tag, userpage banner |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 3 month of osu!supporter tag, userpage banner |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 2 month of osu!supporter tag, userpage banner |
+
+## Organisation
+
+The osu!mania Asian's Champions 2019 is run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ![][flag_ID] [chouyaa](https://osu.ppy.sh/users/8404646) |
+| Co-host | ![][flag_PH] [vincent4399](https://osu.ppy.sh/users/9764388) |
+| Map Selector | ![][flag_ID] [Virtue-](https://osu.ppy.sh/users/6144772), ![][flag_TH] [RuleBlazing](https://osu.ppy.sh/users/7312402), ![][flag_SG] [riunosk](https://osu.ppy.sh/users/5594381) |
+| Streamer | ![][flag_DE] [DinX](https://osu.ppy.sh/users/568111), ![][flag_NL] [NightNarumi](https://osu.ppy.sh/users/4381142), ![][flag_LV] [Adiopulse](https://osu.ppy.sh/users/4387852) |
+| Commentator | ![][flag_VN] [Weabonie](https://osu.ppy.sh/users/8151986), ![][flag_TH] [HowToPlayLN](https://osu.ppy.sh/users/10879600) |
+| Referee | ![][flag_TH] [TNKSTC](https://osu.ppy.sh/users/6037121), ![][flag_ID] [Noch Einen](https://osu.ppy.sh/users/12668735), ![][flag_ID] [FadillSan](https://osu.ppy.sh/users/7006830), ![][flag_ID] [RiP46](https://osu.ppy.sh/users/5222596), ![][flag_TW] [Makura](https://osu.ppy.sh/users/10665623), ![][flag_FR] [Mathisca](https://osu.ppy.sh/users/2227462) |
+| Statistician | ![][flag_TH] [mashedpotato_old](https://osu.ppy.sh/users/496237) |
+| Recap Editor | ![][flag_KR] [Misuku](https://osu.ppy.sh/users/4799593) |
+| Wiki | ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/946117)
+- [Challonge](https://challonge.com/zh_CN/asianschampions2019)
+- [Livestream](https://www.twitch.tv/asianschampions)
+- **Statistics sheet**
+  - [Mappool](https://docs.google.com/spreadsheets/d/1aGQHz8YRl-8a8mdp-7E5h_KQBk3rnwlvUzePaIrx6Zs)
+  - [Schedule](https://docs.google.com/spreadsheets/d/1vbSfQXcCG-CMeBmwlCKN5WW1gaaY48yqGsjyFvcN0zU)
+
+## Mappools
+
+### Group Stage
+
+- FreeMod
+  - [Tennyson - No Answer \[Silence\]](https://osu.ppy.sh/beatmapsets/977509#mania/2045881)
```

Comment:
map format should go `Artist - Title (Creator) [Diff]`, & don't write Source before this